src/main/java/solrocr/OcrHighlighter.java

```diff
@@ -440,7 +448,7 @@ public OcrHighlightResult[] highlightOcrFields(
                   if (contentFinal.getPointer() != null) {
                     log.error(
                         "Could not highlight OCR content for document {} at '{}'",
-                        docIdFinal,
+                        readerDocIdFinal,
```

Comment:
Should we log `readerDocIdFinal`, which may be relative to a given leaf reader, or `indexDocId`, which is the index-wide docId?
The same applies to line 463.